dynamodb_revisions/add_merge.py

```diff
@@ -0,0 +1,54 @@
+"""Add merged records and references to database."""
+from therapy.database import Database
+from therapy.etl.merge import Merge
+from timeit import default_timer as timer
+from therapy.schemas import SourceName
+from therapy import PROJECT_ROOT
+
+
+def main():
+    """Execute scan on DB to gather concept IDs and add merged concepts."""
+    start = timer()
+    db = Database(db_url='http://localhost:8000')
+
+    concept_ids_path = PROJECT_ROOT / 'dynamodb_revisions' / 'concept_ids.txt'
```

Comment:
👍 I think we can just delete the concept_ids file completely since we shouldn't need to run repeated merge processes on the same IDs anymore (ie no longer testing)